game-core/src/main/java/games/strategy/triplea/delegate/MustFightBattle.java

```diff
@@ -72,10 +72,10 @@
 
   private static final long serialVersionUID = 5879502298361231540L;
   // maps Territory-> units (stores a collection of who is attacking from where, needed for undoing moves)
-  private Map<Territory, Collection<Unit>> m_attackingFromMap = new HashMap<>();
+  private Map<Territory, Collection<Unit>> m_attackingFromMap;
   private final Collection<Unit> m_attackingWaitingToDie = new ArrayList<>();
-  private Set<Territory> m_attackingFrom = new HashSet<>();
-  private Collection<Territory> m_amphibiousAttackFrom = new ArrayList<>();
+  private Set<Territory> m_attackingFrom;
+  private Collection<Territory> m_amphibiousAttackFrom;
```

Comment:
Would it make more sense to get rid of the superfluous assignments in the constructor instead of the initializers?

Reply:
I don't have a strong opinion on that. Sounds like we should add that preference to the code standard.